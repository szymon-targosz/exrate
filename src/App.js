import React, { Component } from 'react';
import DefineRate from './components/DefineRate';
import AddTransaction from './components/AddTransaction';
import Transactions from './components/Transactions';
import HighestTransaction from './components/HighestTransaction';
import Header from './components/Header';
import Filters from './components/Filters';

export default class App extends Component {
   render() {
      return (
         <React.Fragment>
            <Header />
            <section className="content container">
               <DefineRate />
               <AddTransaction />
               <HighestTransaction />
               <Filters />
               <Transactions />
            </section>
         </React.Fragment>
      );
   }
}
