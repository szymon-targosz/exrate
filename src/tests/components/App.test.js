import React from 'react';
import App from '../../App';
import { shallow } from 'enzyme';

describe('COMPONENT - App', () => {
   test('Should render App correctly', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
   });
});
