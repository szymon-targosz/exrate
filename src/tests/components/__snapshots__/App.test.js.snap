// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - App Should render App correctly 1`] = `
<Fragment>
  <Header />
  <section
    className="content container"
  >
    <Connect(DefineRate) />
    <Connect(AddTransaction) />
    <Connect(HighestTransaction) />
    <Connect(Filters) />
    <Connect(Transactions) />
  </section>
</Fragment>
`;
